### PR TITLE
r-rcppparallel updates

### PR DESCRIPTION
**Pull request by bgruening**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Working on https://github.com/bioconda/bioconda-recipes/issues/3340 with @zachcp

### CODE REVIEW

**Comment by zachcp** [comment]
thanks @bgruening!

 I will try building this version; I had used `conda skeleton CRAN` to generate this package.

**Comment by bgruening** [comment]
@zachcp can you add `-headerpad_max_install_names` to the ld flags if compiling under OSX.

**Comment by zachcp** [comment]
HI @bgreuning.  I've tried a few variants of passing in -headerpad_max_install_names to no avail.  I am going to ping the Rcpp-prallel team for suggestions.

```
$R CMD INSTALL --build  --configure-args="-headerpad_max_install_names"  .

$R CMD INSTALL --build  --configure-args="LDFLAFS-headerpad_max_install_names"  .
```

**Comment by zachcp** [comment]
@bgruening 

Looks like we may need to patch the buildfile for the TBB library. Do you know of any examples of patches of build files that I could work from?  https://github.com/RcppCore/RcppParallel/issues/40#issuecomment-270770718

**Comment by bgruening** [comment]
@zachcp you can use real patch files like here: https://github.com/bioconda/bioconda-recipes/blob/f5eb63e30a76fd13c28663786d219c9f7750267c/recipes/detonate/meta.yaml#L34

or you can simply use `sed` like here: https://github.com/bioconda/bioconda-recipes/blob/c965a99e74ec52a7c33eb5cb14f8fbdc744aa6f8/recipes/kronik/build.sh#L3

for changing a single line.

**Comment by zachcp** [comment]
this now builds locally so hopefully this will pass travis. thanks @bgruening !

File: recipes/r-rcppparallel/meta.yaml [hunk 1]
```diff
@@ -13,48 +8,27 @@ source:
...
-   # List any patch files here
-   # - fix.patch
-
 build:
-  # If this is a new build for the same version, increment the build
-  # number. If you do not include this key, it defaults to 0.
-  # number: 1
-
-  # This is required to make R link correctly on Linux.
+  number: 0
```
Comment (bgruening) [inline recipes/r-rcppparallel/meta.yaml]: One last thing, you need to increase the version number, otherwise nothing will be tested.

**Comment by zachcp** [comment]
good to go?

**Comment by bgruening** [comment]
Yeah! Thanks a lot!

**Comment by zachcp** [comment]
thanks for your help/guidance @bgruening !

**Comment by bgruening** [comment]
Sure, anytime. Thanks for your contribution!

**Pull request merged**